metrics-delivery-endpoint-impl/src/main/java/eu/openaire/mas/delivery/provider/PrometheusMetricsProvider.java

```diff
@@ -53,7 +53,8 @@ public MetricEntry deliver(String groupId, String metricId, String from, String
                     // FIXME it is just a sample, include "from" and "to" params in querying
                     VectorResponse resp = prometheusClient.query(meta.getQuery());
                     if (STATUS_SUCCESS.equals(resp.getStatus())) {
-                        return new MetricEntry(groupId, metricId, resp.getData());    
+			float value = resp.getData().getResult().get(0).getValue().get(1);
```

Comment:
Does `STATUS_SUCCESS` guarantee we won't get NPEx when digging through `get(0).getValue().get(1)` path? We might want to check for nulls here just to avoid propagating nasty NPEx to the client.

Btw, is the value we want to provide always stored as 2nd element among the collection of values (as returned by `getValue()` method)? What is the 1st element then?